src/components/HistoryMenuBtn/HistoryMenuBtn.jsx

```diff
@@ -0,0 +1,15 @@
+import { useVideoContext, useAuthContext } from "../../context/context";
```

Comment:
appreciation : i liked you add to this component for history
addition : you need to put a clear history btn as per prd

Reply:
Thanks will add the button.